dd-sdk-android/src/main/kotlin/com/datadog/android/webview/internal/rum/WebViewRumEventConsumer.kt

```diff
@@ -29,28 +30,29 @@ internal class WebViewRumEventConsumer(
         GlobalRum.notifyIngestedWebViewEvent()
         val rumContext = contextProvider.getRumContext()
         val mappedEvent = map(event, rumContext)
-        if (mappedEvent != null) {
-            dataWriter.write(mappedEvent)
-        }
+        dataWriter.write(mappedEvent)
     }
 
     private fun map(
         event: JsonObject,
         rumContext: RumContext?
-    ): JsonObject? {
+    ): JsonObject {
         return try {
```

Comment:
Same thing, the return value is always the same, so this syntax makes it harder to read.